test/e2e-test/nextjs.test.ts

```diff
@@ -0,0 +1,120 @@
+import util = require('node:util');
+// eslint-disable-next-line @typescript-eslint/no-var-requires, no-undef
+const execFile = util.promisify(require('node:child_process').exec);
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const copy = require('recursive-copy');
+
+import AWS = require('aws-sdk');
+
+import { expect } from 'chai';
+
+describe('Create nextjs application', () => {
+    let templatePath: string;
+    let template: string;
+    let realPath: string;
+    let localOutput: string;
+    const bucketParams = {
+        Bucket: "azion-test"
+    }
+
+    const s3 = new AWS.S3 ({
+        accessKeyId: "123456",
+        secretAccessKey: "123456",
+        signatureVersion: "v4",
+        s3ForcePathStyle: true,
+        endpoint: "http://localhost:4566"
+    });
+
+    before(async () => {
+        // Creates temporary local template repository
+        templatePath = fs.mkdtempSync(path.join(os.tmpdir(), 'nextjs-test'));
+        realPath = fs.realpathSync(templatePath);
+        const templateName = 'nextjs-template';
+        template = path.join(realPath, templateName);
+        localOutput = process.cwd();
+
+        await s3.createBucket(bucketParams).promise();
+    });
+
+    after(() => {
+        process.chdir(localOutput);
+        if (templatePath) {
+            fs.rmSync(templatePath, { recursive: true });
+        }
+    });
+
+    it('init template nextjs"', async () => {
+        console.log('Creating Next project');
+        await execFile(`npx -y create-next-app@latest ${template}`);
+        const data = fs.readFileSync(`${template}/pages/index.js`);
+        const dataToString = data.toString();
+        const dataToArray = dataToString.split('\n');
+        dataToArray.splice(1,1); //Remove import of NextJS/Image
+        dataToArray.splice(61,1,'               <img src="/vercel.svg" alt="Vercel Logo" width="72" height="16" />\n');
+        const arrayToText =  dataToArray.join('\n');
+        fs.writeFileSync(`${template}/pages/index.js`,arrayToText)
+        process.chdir(template);
+        await execFile(`npx next build`);
+        await execFile(`npx next export`);
+    });
```

Comment:
Shouldn't we see at least one `expect` here? Perhaps a test on the `execFile` return code?